_episodes/02-day1.md

```diff
@@ -19,7 +19,72 @@ keypoints:
 - "Vector data structures represent specific features on the Earth's surface along with attributes of those features."
 - "Vector objects are either points, lines, or polygons."
 ---
-FIXME
+This episode reviews the two primary types of geospatial data: rasters and vectors. It will provide an overview of these two types of geospatial data, describing major features of both and providing examples of where you might find each type. 
+
+# Data Structures: Raster and Vector
+
+Raster data is any pixelated (or gridded) data where each pixel is associated with a specific geographic location. The value of a pixel can be continuous (e.g. elevation) or categorical (e.g. land use). This data structure is very common: it's how we represent any digital image. A geospatial raster is only different from a digital photo in that it is accompanied by spatial information that connects the data to a particular location. This includes the raster's extent and cell size, the number of rows and columns, and its coordinate reference system (CRS). 
+
+Some examples of continuous rasters include:
+
+1. Precipitation maps
+2. Maps of tree height derived from LiDAR data 
+3. Elevation values for a region
+
+Some rasters contain categorical data where each pixel represents a discrete class such as landcover type (e.g., "forest" or "grassland") rather than a continuous value such as elevation or temperature. Some examples of classified maps include:
```

Comment:
I've always heard this described as "discrete" data in geospatial contexts, but calling it categorical seems to be more common in general statistics. If it comes up in later lessons at all, we should be consistent!